README.Rmd

```diff
@@ -44,7 +44,8 @@ The following methodology/features are currently implemented:
 -   Accounting for feature dependence assuming the features are Gaussian (@aas2019explaining).
 -   Accounting for feature dependence with a Gaussian copula (Gaussian dependence structure, any marginal) (@aas2019explaining).
 -   Accounting for feature dependence using the Mahalanobis distance based empirical (conditional) distribution approach of @aas2019explaining.
--   Combine any of the three methods.
+-   Accounting for feature dependence using conditional inference trees (@hothorn2006unbiased). This methodology has currently been submitted. 
```

Comment:
What does "This methodology has currently been submitted." mean?

Reply:
Updated readme with ctree paper reference.

Reply:
Great.